### PR TITLE
GEOPY-889: fix for geoh5py release 0.7.0

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-889 - Fix for geoh5py release 0.7.0**

### CODE REVIEW

File: Install_or_Update.bat
Comment (sebhmg) [inline Install_or_Update.bat:1]: out of scope: addresed in https://github.com/MiraGeoscience/geoapps/pull/522

File: geoapps/inversion/components/factories/entity_factory.py
Comment (sebhmg) [inline geoapps/inversion/components/factories/entity_factory.py:1]: **suggestion (tests):** update unit tests

**Review by sebhmg — COMMENTED** [review]
let's wait for the following PRs to be merged:
- https://github.com/MiraGeoscience/geoapps/pull/522
- https://github.com/MiraGeoscience/geoapps/pull/520

File: geoapps/inversion/components/factories/entity_factory.py
Comment (domfournier) [reply geoapps/inversion/components/factories/entity_factory.py:1]: unittested on geoh5py. That's where it should error.

**Comment by sebhmg** [comment]
merged into https://github.com/MiraGeoscience/geoapps/pull/524

**Pull request closed without merging**